pkg/api/router.go

```diff
@@ -336,6 +336,12 @@ func (s *Service) mountAPI() {
 		web.FinalHandlerFunc(s.healthHandler),
 	))
 
+	handle("/rchash/{depth}/{anchor1}/{anchor2}", web.ChainHandlers(
```

Comment:
don't forget to add this endpoint to the list in `auth.go`

Reply:
@notanatol what should be right policy for this endpoint?